From 616d5d60572ba57fad45aa139a99eb6a43a73c5f Mon Sep 17 00:00:00 2001
From: Mladen Milinkovic <maxrd2@smoothware.net>
Date: Thu, 1 Aug 2024 10:56:02 +0200
Subject: [PATCH 06/11] VideoPlayer: stop injecting stream side data in packets

This is no longer needed as the side data is available for decoders in the
AVCodecContext.
---
 src/videoplayer/backend/streamdemuxer.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/videoplayer/backend/streamdemuxer.cpp b/src/videoplayer/backend/streamdemuxer.cpp
index 9c949fa8..eda2b267 100644
--- a/src/videoplayer/backend/streamdemuxer.cpp
+++ b/src/videoplayer/backend/streamdemuxer.cpp
@@ -501,8 +501,6 @@ StreamDemuxer::run()
 	if(m_vs->genpts)
 		ic->flags |= AVFMT_FLAG_GENPTS;
 
-	av_format_inject_global_side_data(ic);
-
 	{ // find_stream_info
 		const int origNbStreams = ic->nb_streams;
 		AVDictionary **opts = (AVDictionary **)av_calloc(origNbStreams, sizeof(*opts));
-- 
2.46.0

